src/ports/postgres/modules/graph/hits.sql_in

```diff
@@ -164,10 +168,20 @@ INSERT INTO edge VALUES
 (3, 0, 1),
 (4, 0, 1),
 (5, 6, 1),
-(6, 3, 1);
+(6, 3, 1),
+(0, 1, 2),
+(0, 2, 2),
+(0, 4, 2),
+(1, 2, 2),
+(1, 3, 2),
+(2, 3, 2),
+(3, 0, 2),
+(4, 0, 2),
+(5, 6, 2),
+(6, 3, 2);
 </pre>
 
--# Compute the HITS scores:
+-# Running HITS with default values for optional parameters:
```

Comment:
In those examples without grouping, duplicate edges from different groups are calculated twice. We discussed this before and just wondering if we can add some explanation here(may not be necessary tho).

Reply:
@jingyimei good point. we checked in a commit to remove the extra user_id `2` from the edge table when hits is called without the grouping column.